### ISSUE TITLE
Sorting on Product/Order/Discount dashboard pages

### ISSUE THREAD

**Issue by Mesuva** [VividWeb/vivid_store#151]
As a future enhancement it would be nice if the columns on these dashboard pages actually did sort the data displayed.

**Comment by brucewyne** [comment]
Yeah - this should be fairly easy. I'm going to group this with 3.0

**Comment by brucewyne** [comment]
Meh - I'm going to change the milestone to next release (post 3.0)
